src/main/java/com/offdk/play/model/SlackCommand.java

```diff
@@ -5,9 +5,12 @@
   private String teamId;
   private String teamDomain;
   private String channelId;
+  private String channelName;
   private User commandUser;
+  private String command;
   private String text;
   private String responseUrl;
+
```

Comment:
Actually, on second thought, does it make sense to use the builder construct here? AutoValue really wasn't that bad as long as you werent trying to do anything too crazy? And it does make it a bit cleaner when testing.